packages/webapp/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "Serves a Meteor app over HTTP",
-  version: '1.3.16'
+  version: '1.3.17'
 });
 
 Npm.depends({connect: "2.30.2",
```

Comment:
Not related to these changes, but we should really consider jumping to `connect` 3.x at some point.